tensorboard/webapp/plugins/npmi/views/data_selection/results_download/results_download_test.ts

```diff
@@ -0,0 +1,84 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
If the `metricIsNpmiAndNotDiff` in results_download_container is an important piece of business logic, it might also be good to have a test case for that as well

Reply:
Yes, agreed. This is part of another PR that I requested Stephans review for where I implemented tests for all the remaining util functions.